docs/user/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -15,12 +15,21 @@ image::user/monitoring/images/metricbeat.png[Example monitoring architecture]
 To learn about monitoring in general, see
 {ref}/monitor-elasticsearch-cluster.html[Monitor a cluster].
 
-//NOTE: The tagged regions are re-used in the Stack Overview.
+****
+**Question for reviewers:**
+
+Is it still necessary to disable default collection? I didn't test this with
+Metricbeat, but when I tested Elastic Agent + the Kibana integration, I did not
+need to change any default settings. I did not need to set
+monitoring.kibana.collection.enabled: false, and I didn't see
+a monitoring.enabled setting.
```

Comment:
I'm not sure what these settings are enabling, but setting it to true or false didn't appear to change anything. I'll dig a bit further

Reply:
@klacabane Should I just remove the step about disabling default monitoring collection? Or just leave it here since the focus of this PR isn't really updating the Metricbeat-related steps?

@lcawl I don't think these tagged regions are used anymore right?